### ISSUE TITLE
Implement the new \expanded pdfTeX primitive

### ISSUE THREAD

**Issue by dginev** [brucemiller/LaTeXML#1189]
Announcement+Documentation:
https://www.texdev.net/2018/12/06/a-new-primitive-expanded

Thanks to @wtsnjp for brainstorming creative solutions to a "rosetta stone" type challenge [on twitter](https://twitter.com/wtsnjp/status/1162533459656499201), I realized latexml doesn't yet support the new primitive, which is bound to cause hard to diagnose problems. 

This is related to / part of getting to latex 3 support ( #1082 ), as one can spot in the definitions:
```tex
\__kernel_primitive:NN \expanded              \tex_expanded:D
```
so it is already somewhat expected, though optionally so, there are fallbacks. It's a very handy primitive to have around, so might as well give it some attention.

**Comment by brucemiller** [comment]
Cool! First thing to notice was that running the test case in the above link produced nonsense with no errors, but after pulling the recent PR, happily throws an Error (```\expanded``` not defined). So *that* is a good thing after all :>

Second thing to notice is that a trivial ```DefMacro('\expanded Expanded', '#1');```  works on that test case.  Is that all there is???

**Comment by dginev** [comment]
Could be! The benefits of implementing handy high level paremeter types for our own internal convenience, only to realize that the pdfTeX people ended up working around the same set of pain and 
suffering :> You should really try the computational example, which should keep latexml's wheels spinning for a bit.

https://gist.github.com/wtsnjp/5a3bfe56a6278ac1451d73394c1ee2ba

**Comment by brucemiller** [comment]
I don't have a working luatex, so I don't know what it should produce, but after 22sec, I get 2024955000.  Is that close?

**Comment by dginev** [comment]
`2024955000` is indeed the answer by `pdfTeX, Version 3.14159265-2.6-1.40.19 (TeX Live 2019/dev/Debian)`. Takes the eternity of 0.2 seconds! So looks like you have it solved :+1:

**Comment by brucemiller** [comment]
Now that I have a working luatex, the first sample turns up a curious oversight: the doubling of "#" under certain circumstances (passing of # within arguments? googled explanations vary)

**Comment by dginev** [comment]
I never actually ran luatex, just a recent `pdftex`. Since latexml is trying to pretend to be `pdftex`/`pdflatex` under most circumstances, it would be helpful to understand how the luatex and pdftex versions vary, if at all.

**Comment by brucemiller** [comment]
Maybe; I didn't track down versions, but assumed I
didn't have a new enough pdftex. The thread you cited
said that luatex already had the feature (since it
was forked from a dev version of pdftex)

**Comment by dginev** [comment]
closed via 4ab7dc15e51b5c2db9d491bfe67037887248d917
